src/bmark_solvers.jl

```diff
@@ -13,13 +13,27 @@ Run a set of solvers on a set of problems.
 Any keyword argument accepted by `solve_problems`
 
 #### Return value
-A Dict{Symbol, AbstractExecutionStats} of statistics.
+A Dict{Symbol, DataFrame} of statistics.
 """
-function bmark_solvers(solvers::Dict{Symbol, <:Any}, args...; kwargs...)
+function bmark_solvers(solvers::Dict{Symbol, <:Any}, args...;parallel_eval=false, kwargs...)
```

Comment:
is this mandatory? what happens if we try to run the benchmarks in parallel and there is only 1 proc?

Reply:
it's mandatory to set `parallel_eval=true` if you want to benchmark in parallel. If you still have 1 proc, it won't do it in parallel. I added this keyword in case the user is handling multiple procs and does not want evaluate solvers in parallel. We could get rid of it it's not necessary.

Reply:
Ok we could leave kt, at least for debugging, but let's set it to `true` by default.